src/js/app.js

```diff
@@ -36,16 +37,58 @@ const app = {
     });
   },
 
+  initPages: function(){
+    const thisApp = this;
+    thisApp.pages = Array.from(document.querySelector(select.containerOf.pages).children);
+    thisApp.navLinks = Array.from(document.querySelectorAll(select.nav.links));
+    let pagesMatchingHash = [];
+    if (window.location.hash > 2){
+      const idFromHash = window.location.hash.replace('#/', '');
+      pagesMatchingHash = thisApp.pages.filter(function(page){
+        return page.id == idFromHash;
+      });
+      thisApp.activatePage(pagesMatchingHash.length ? pagesMatchingHash[0].id : thisApp.pages[0].id);
```

Comment:
przenieś linię 50 niżej, poza ifa, ponieważ ona sprawdza, czy mamy coś w zmiennej dopasowującej stronę do hasha, a jeśli nie (czyli u nas localhost:3000 - bez hasha), to wyświetl pierwszą ze stron, które mamy w kodzie (thisApp.pages[0].id).

If w linii 45 sprawdza, czy mamy coś w hashu więc jesli jest pusto, to nawet nie wejdzie do tego ifa i tej linii nie odpali

Reply:
Hej,
zmieniłem, teraz jak odpala się stronę, to już wyświetla się zakładka "Orders". Czy to nie miało działać na zasadzie, że jak jestem w zakładce "Bookings" i odświeżam stronę to pozostaje na tej zakładce?

Reply:
Dokladnie. Powinno byc tak, ze domyslnie na `localhost:3000` laduja sie `orders`, a jak bedziemy na `booking` i odswiezymy, to nadal jestesmy na booking - jakies pomysly?